Automattic/wp-calypso
# Split up lib/formatting

This PR splits up `lib/formatting` from a single monolithic file, to an index that re-exports individual functions. This allows `webpack` to tree-shake the library more effectively.

#### Changes proposed in this Pull Request

* Split up `lib/formatting`

#### Testing instructions

There are no significant code-changes, only moving things around. The unit tests and a successful build should be enough to verify that everything still works correctly.
